core/templates/dev/head/pages/exploration_player/StatsReportingService.js

```diff
@@ -48,6 +56,12 @@ oppia.factory('StatsReportingService', [
         });
     };
 
+    $rootScope.$on('sessionTime', function(evt, totalTime, stateName) {
```

Comment:
Units... (totalTime)

Reply:
@seanlip I'm sorry, I don't understand this.

Reply:
The totalTime arg needs a name that contains units, similar to what I said above for MIN_TIME_FOR_ACTUAL_START_SECS.

Reply:
Done.